packages/app-elements/src/ui/resources/ResourceList/ResourceList.tsx

```diff
@@ -184,36 +265,51 @@ export function ResourceList<TResource extends ListableResourceType>({
     )
   }
 
-  const isEmptyList = data != null && data.list.length === 0
   if (isEmptyList) {
-    return <>{emptyState}</>
+    return variant === 'table' ? (
```

Comment:
As far as I remember we discussed to have this one as default behavior in case the `emptyState` is a string (eventually the default one based on the resource name). Am I wrong?